tests/compute/test_transform.py

```diff
@@ -261,6 +261,10 @@ def test_metis_partition():
         assert num_inner_nodes == g.number_of_nodes()
         print(g.number_of_edges() - num_inner_edges)
 
+def test_partition():
+    test_metis_partition()
+    test_partition_with_halo()
```

Comment:
`test_blah` should be automatically tested by unit test modules; you don't have to add this function.

Reply:
OK, I have reverted it.